akka/shopping-cart-java/docker-compose.yml

```diff
@@ -33,8 +33,6 @@ services:
     environment:
       KAFKA_ADVERTISED_HOST_NAME: localhost
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Just curious: why was that removed?

Reply:
The samples here are synced from the Akka Guide samples. You can check the history there:

https://github.com/akka/akka-guide

Reply:
For example: https://github.com/akka/akka-guide/pull/711